src/core/Chip/styles.css

```diff
@@ -0,0 +1,85 @@
+.container {
+    align-items: center;
+    background-color: #eceff1;
+    border-radius: 16px;
+    cursor: pointer;
+    display: inline-flex;
+    font-size: 14px;
+    height: 32px;
+    margin: 4px;
+    user-select: none;
+}
+
+.container:hover {
+    background-color: #e0f2f1;
+}
+
+.disabled {
+    cursor: not-allowed;
+    opacity: 0.3;
+}
+
+.disabled .remove-icon {
+    pointer-events: none;
+}
+
+.dragging {
+    box-shadow: 0 3px 1px -2px rgba(0, 0, 0, 0.2),
+        0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 1px 5px 0 rgba(0, 0, 0, 0.12);
+}
+
+.icon {
+    color: #494949;
+    font-size: 20px;
+    margin-left: 5px;
+}
+
+.image-icon {
+    border-radius: 50%;
+    height: 24px;
+    margin-left: 4px;
+    width: 24px;
+}
+
+.label {
+    margin: 0 12px;
+    max-width: 150px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+.remove-icon {
+    color: #494949;
+    cursor: pointer;
+    font-size: 18px;
+    margin-right: 4px;
+    opacity: 1;
+    pointer-events: all;
+}
+
+.remove-icon:hover {
+    opacity: 0.82;
+}
+
+.selected {
+    background-color: #00796b;
+}
+
+.selected:hover {
```

Comment:
move `.selected` and `selected:hover` class relative to the `.container` at the top